app/code/core/Mage/Adminhtml/Block/Widget/Grid/Column/Renderer/Abstract.php

```diff
@@ -115,7 +115,7 @@ protected function _getInputValue(Varien_Object $row)
     /**
      * @return string
      */
-    public function renderHeader()
+    public function renderHeader(): string
```

Comment:
doesn't modifying the signature of public method break every module that overrides this method?

Reply:
Indeed, I forgot to remove that yesterday